lib/webauthn/authenticator_assertion_response.rb

```diff
@@ -12,10 +12,10 @@ def initialize(credential_id:, authenticator_data:, signature:, **options)
       @signature = signature
     end
 
-    def valid?(original_challenge, original_origin, allowed_credential:)
+    def valid?(original_challenge, original_origin, allowed_credentials:)
       super(original_challenge, original_origin) &&
-        valid_credential?(allowed_credential[:id]) &&
-        valid_signature?(allowed_credential[:public_key])
+        valid_credentials?(allowed_credentials) &&
```

Comment:
We are actually validating only one credential against a set of allowed credentials. Wouldn't that mean that this line should be `valid_credential?(allowed_credentials)`?